src/components/SelectBox.resi

```diff
@@ -106,13 +196,13 @@ module BaseRadio: {
     ~deselectDisable: bool=?,
     ~onBlur: ReactEvent.Focus.t => unit=?,
     ~fill: string=?,
-    ~customStyle: Js.String2.t=?,
+    ~customStyle: string=?,
     ~searchable: bool=?,
     ~isMobileView: bool=?,
     ~customSearchStyle: string=?,
     ~descriptionOnHover: bool=?,
     ~addDynamicValue: bool=?,
-    ~dropdownCustomWidth: Js.String2.t=?,
+    ~dropdownCustomWidth: string=?,
```

Comment:
These changes in SelectBox types can be moved to seperate PR?

Reply:
Because of Window changes these have come. So it should be in this PR.